UI/ChatSignalR/UnoChat.Client/UnoChat.Mobile/Android/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.companyname.UnoChat" android:versionCode="1" android:versionName="1.0">
-  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" />
+  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="33" />
```

Comment:
@Marc-Antoine-Soucy I think we can remove this line completely. If you look at the latest unoapp template we don't include this.